.github/workflows/post-apiview.yml

```
@@ -0,0 +1,22 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+
+permissions: {}
+
+jobs:
+  event-handler:
+    permissions:
+      statuses: write
+      pull-requests: write
+      checks: read
+    name: Handle ${{ github.event_name }} ${{ github.event.action }} event
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

Our guideline is to pin ubuntu versions: https://github.com/Azure/azure-sdk-tools/wiki/GitHub-Actions-Design-Guidelines#guidelines